TodoList/Extensions/NSObjectExtensions.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+// Extension: To avoid typing string as it is prone to error. EX: TodoListViewController.className
+extension NSObject {
+    var className: String {
+        return String(describing: type(of: self))
+    }
+    
+    class var className: String {
+        return String(describing: self)
+    }
```

Comment:
Love this. I just wanted to know what's the difference between doing `class var` and `static let`

Reply:
since its an extension, it needs to be a computer property.

Reply:
I found my answer here: https://stackoverflow.com/questions/29636633/static-vs-class-functions-variables-in-swift-classes
> `static` and `class` both associate a method with a class, rather than an instance of a `class`. The difference is that subclasses can override `class` methods; they cannot override `static` methods

Given this info I would do:
```
static var className: String {
        return String(describing: self)
}
```

Reply:
I thought you meant static let... it makes sense to switch to static var👍

Reply:
Yeah, I did mean static let initially and you were right about computed properties, but static var is still possible. I'm loving this!